javascript-stringify.js

```diff
@@ -229,6 +229,11 @@
       return PRIMITIVE_TYPES[typeof value](value, indent, next);
     }
 
+    // Handle Error objects.
+    if (value instanceof Error) {
```

Comment:
@zalmoxisus Yep, I realise - but at least it works across frames. Possibly all the code could move to `instanceof` checks instead, but since performance isn't the primary concern for the library I haven't looked into that refactor.